crates/api_models/src/payments.rs

```diff
@@ -5171,6 +5172,14 @@ pub struct PaymentsConfirmIntentResponse {
 
     /// Error details for the payment if any
     pub error: Option<ErrorDetails>,
+
+    /// The transaction authentication can be set to undergo payer authentication. By default, the authentication will be marked as NO_THREE_DS
+    #[schema(value_type = Option<AuthenticationType>, example = "no_three_ds")]
+    pub authentication_type: Option<api_enums::AuthenticationType>,
+
+    /// The authentication type applied for the payment
+    #[schema(value_type = AuthenticationType, example = "no_three_ds", default = "no_three_ds")]
+    pub applied_authentication_type: api_enums::AuthenticationType,
```

Comment:
Discussed this offline with @Narayanbhat166, we can remove the `default` in the response type.